src/test/java/com/fryrank/controller/MessageControllerTests.java

```diff
@@ -0,0 +1,24 @@
+package com.fryrank.controller;
+;
+
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.junit.Assert;
+
+import static com.fryrank.Constants.WELCOME_MESSAGE;
+
+public class MessageControllerTests {
+
+    @Autowired
+    private MessageController controller;
+
+    @Test
+    public void testGetWelcomeMessage() throws Exception {
+        MessageController controller = new MessageController();
```

Comment:
Remove this

Reply:
Done in b1cd8234a3db8d047ba0a0127860fc45e407be8a